public/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html ng-app="todoListApp">
+  <head>
+    <title>Things ToDo</title>
+    <script type="text/javascript" src="assets/angular/angular.min.js"></script>
+    <script type="text/javascript" src="assets/jquery/dist/jquery.min.js"></script>
+    <script type="text/javascript" src="js/app.js"></script>
+    <link href="style.css" rel="stylesheet">
+  </head>
+
+  <body>
+
+    <h1>Todo Today</h1>
+    <main>
+
+      <section ng-controller="TodoList as list">
+
+        <form name="addItem" ng-submit="addItem.$valid && list.addItem()" novalidate>
+
+          <input type="text" ng-model="list.newItem.content" required />
+          <br />
+          <input type="submit" />
+        </form>
+        <article ng-show="list.newItem.content">
+          {{list.newItem.content}}
+        </article>
+        <article ng-repeat="item in list.items">
+          <span ng-class="{completed : item.isCompleted}">{{item.content}}</span>
+
+        </article>
+
+      </section>
+
+
+
+    </main>
+
+
+
+
+  </body>
+
+
+
+</html>
```

Comment:
Whitespace! Fine when you're developing, but nice to have tidied up before submitting